modules/virt-connecting-service-ssh.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-accessing-vm-ssh.adoc
+// * virt/virtual_machines/vm_networking/virt-creating-service-vm.adoc
+
+:_content-type: PROCEDURE
+[id="virt-connecting-service-ssh_{context}"]
+= Connecting to a virtual machine by using SSH and a service
+
+You connect to a virtual machine (VM) by using SSH and a Kubernetes service.
+
+.Prerequisites
+
+* You created a Kubernetes service to expose a VM.
+* You have an SSH client installed.
+
+.Procedure
+
+* For a `ClusterIP` service, specify the service IP address and the service port:
++
+[source,terminal]
+----
+$ ssh <user>@<cluster_IP> -p <port>
+----
+
+* For a `NodePort` service, specify the node IP address and the node port:
++
+[source,terminal]
+----
+$ ssh fedora@$NODE_IP -p 30000
+----
+
+* For a `LoadBalancer` service, use the `vinagre` client to connect to your virtual machine by specifying the public IP address and port. External ports are dynamically allocated.
```

Comment:
@dominikholler I discovered that Vinagre is obsolete. Can you suggest a different client that uses the load balancer service?